src/commons/controlBar/ControlBarStepLimit.tsx

```diff
@@ -15,6 +16,16 @@ type StateProps = {
 };
 
 export const ControlBarStepLimit: React.FC<ControlBarStepLimitProps> = props => {
+  /**
+   * Scale the odd stepLimit into the next even integer.
+   * Triggered when the input box lost focus.
+   */
+  const onBlurAutoScale = () => {
+    props.handleOnBlurAutoScale !== undefined
+      ? props.handleOnBlurAutoScale(Number(props.stepLimit))
+      : console.log('undefined');
```

Comment:
Not sure about this console.log: Is it for your own debugging? The console log "undefined" isn't helpful for the user (or developer). Can you take another look?

Reply:
<img width="927" alt="image" src="https://user-images.githubusercontent.com/60089102/218019372-345e9a2f-efaf-46b9-b454-b1ef2f92ab8e.png">
I changed it to use if statement. Is it okey to write like that?

Reply:
a simpler way to do this could be via [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

```javascript
props.handleOnBlurAutoScale?.(...)
```

Reply:
Okey! Thank you very much. I will change it to optional chaining.

Reply:
<img width="850" alt="image" src="https://user-images.githubusercontent.com/60089102/218235764-e486ae22-8641-4dd9-a441-0c6a0e7e6799.png">
changes to optional chaining committed.